src/constants/functionKey.ts

```diff
@@ -0,0 +1,27 @@
+enum FunctionKey {
+  // Queries
+  GET_MARKETS = 'GET_MARKETS',
+  GET_ASSETS_IN_ACCOUNT = 'GET_ASSETS_IN_ACCOUNT',
+  GET_VTOKEN_BALANCES_ACCOUNT = 'GET_VTOKEN_BALANCES_ACCOUNT',
+  GET_HYPOTHETICAL_LIQUIDITY = 'GET_HYPOTHETICAL_LIQUIDITY',
+
+  // Mutations
+}
+
+export const createGetHypotheticalLiquidityKey = ({
+  account,
+  token,
+}: {
+  account: string | null | undefined;
+  token: string;
+}) => `${FunctionKey.GET_HYPOTHETICAL_LIQUIDITY}-${account}-${token}`;
```

Comment:
Just a heads up that arrays are supported by React Query. That can make things a bit simpler later if we want to retrieve the results from the cache:
```
[FunctionKey.GET_HYPOTHETICAL_LIQUIDITY, token]
```

I also didn't include the account, because the cache is relevant for the connected user only anyway and this query would get wiped out on log out.